CONTRIBUTING.rst

```diff
@@ -0,0 +1,80 @@
+How to contribute
+=================
+
+All contributions are greatly appreciated!
+
+
+Setting up your development environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-   Fork pytest-xprocess to your GitHub account by clicking the `Fork`_ button.
+-   `Clone`_ the main repository (not your fork) to your local machine.
+
+    .. code-block:: text
+
+        $ git clone https://github.com/pytest-dev/pytest-xprocess
+        $ cd pytest-xprocess
+
+-   Add your fork as a remote to push your contributions.Replace
+    ``{username}`` with your username.
+
+    .. code-block:: text
+
+        git remote add fork https://github.com/{username}/pytest-xprocess
+
+-   Create a new virtualenv.
```

Comment:
We actually use tox, so the proper approach here to install xprocess in editable mode with development dependencies is:

```
$ tox --devenv .env
$ . .env/bin/activate
```

Reply:
I ended up creating an isolated [testenv:dev] so it's easier to control things like `basepython` and `usedevelop` specifically for the dev environment. This way, if a contributor only wants to run the basic test suite with `pytest` it will default to latest supported python version and not to whatever version the person happens to be running on their envs. Also, I have just realised I had actually never used tox for doing that, thanks for the tip.